package/yast2-mail.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Nov 30 06:38:15 UTC 2017 - rbrown@suse.com
+
+- Replace references to /var/adm/fillup-templates with new
+  %_fillupdir macro (boo#1069468).
```

Comment:
I do not see this change anywhere in code. Don't you missi t?

Reply:
Sorry, the code was done in #27. I will note the reference in the description.

Reply:
Additionally, I've dropped the compatibility macro as master branch won't be used in older SUSE/openSUSE versions.